litmus-portal/graphql-server/pkg/usermanagement/create.go

```diff
@@ -53,8 +57,7 @@ func CreateUser(ctx context.Context, user model.CreateUserInput) (*model.User, e
 	outputUser = newUser.GetOutputUser()
 	outputUser.Projects = append(outputUser.Projects, project)
 
-	active := os.Getenv("SELF_CLUSTER")
-	if strings.ToLower(active) == "true" && strings.ToLower(outputUser.Username) == "admin" {
+	if strings.ToLower(active) == "true" && strings.ToLower(outputUser.Username) == "admin" && strings.ToLower(agentScope) == "cluster" {
```

Comment:
For namespaced scope self-cluster will not work?

Reply:
Right, it should be for namespace scope also. Will remove the condition